media/css/products/vpn/resource-center.scss

```diff
@@ -50,12 +50,7 @@ $image-path: '/media/protocol/img';
 }
 
 .resource-center-wordmark {
-    @include image-replaced;
```

Comment:
Removed VPN as a background image here and just placed it as a class in the HTML page.

Reply:
This is an open issue with Protocol wordmarks and @craigcook details the difficulty of resolving it here: https://github.com/mozilla/protocol/issues/718

**suggestion (blocking):** Add a new media query for mobile screens only `@media screen and (max-width: #{$screen-md}) ` and set `background-position: top center` on the wordmark here. Also worth adding a comment on these style rules to make clear it's a patch fix (similar to https://github.com/mozilla/bedrock/pull/10874/files#diff-31b556a34950073624d0165cb5376477f1ef2eaf575f0d88629abfa776b2df60R13). You don't have to use this solution exactly, but some patch fix to fully center the workmark on mobile is required.

Reply:
Great way to fix this!